CommunicationLayer/CommDeviceControl/UdpMessageForwarder.h

```diff
@@ -25,23 +25,25 @@
 
 #pragma once
 
+#include <SimpleAmqpClient/SimpleAmqpClient.h>
 #include <QUdpSocket>
 #include "CommunicationLayer/CommDeviceControl/I_MessageForwarder.h"
 
+using namespace AmqpClient;
+
 class I_Settings;
 
 class UdpMessageForwarder : public I_MessageForwarder
 {
-    Q_OBJECT
+  Q_OBJECT
 public:
-    UdpMessageForwarder(I_Settings& settings);
-    virtual ~UdpMessageForwarder();
+  UdpMessageForwarder(I_Settings& settings);
+  virtual ~UdpMessageForwarder();
 
 public slots:
-    void forwardData(QByteArray data);
+  void forwardData(QByteArray data);
```

Comment:
Don't change the indentation in this file. 4 spaces is the standard, let's keep it that way.